internal/imodels/ecosystem/ecosystem.go

```diff
@@ -91,6 +91,53 @@ func (p Parsed) String() string {
 	return str
 }
 
+func (p Parsed) IsValid() bool {
+	// Missing ecosystems here would be caught by the "exhaustive" linter
+	switch p.Ecosystem {
+	case osvschema.EcosystemAlmaLinux,
```

Comment:
This looks like we need to update every time when new ecosystem gets added. 
Not for this PR but I am wondering if we could add the check function to the schema repo or make this function to auto scale.